h4/h4.ino

```diff
@@ -1862,9 +1862,8 @@ void updateAsyncTimerSettings() {
   setDir(getAsyncAxis(), dupr > 0);
 
   // dupr can change while we're in async mode, keep updating timer frequency.
-  timerAlarmWrite(async_timer, getTimerLimit(), true);
   // without this timer stops working if already above new limit
-  timerWrite(async_timer, 0);
```

Comment:
Why do you remove this line? timerAlarm() doesn't seem to reset the timer as per docs.

Reply:
Order of statements here is important, we first configure the timer and then reset it's value to 0.

```
  // dupr can change while we're in async mode, keep updating timer frequency.
  timerAlarm(async_timer, getTimerLimit(), true, 0);
  // without this timer stops working if already above new limit
  timerWrite(async_timer, 0);
```

Reply:
It was my mistake.